core/src/main/resources/schema/app/20191029133507_CB-3210_refactor_telemetry_models.sql

```diff
@@ -0,0 +1,24 @@
+-- // CB-3210 refactor telemetry models (v2)
+-- Migration SQL that makes the change goes here.\
```

Comment:
This migration script is going to be a problem. :( 
On the production environment we have zero downtime deployments. It's not possible with this change, because an older version of Cloudbreak won't be able to run with this newer schema. I have to ask you to think through how could you meet the requirements of the zero downtime deployment with this large refactoring.

This wiki page will help:
https://cloudera.atlassian.net/wiki/spaces/ENG/pages/182452362/Database+schema+changes+with+zero+downtime

Reply:
I think this cannot be merged until it's figured out and done.